R/adjective.R

```diff
@@ -51,6 +51,7 @@ adjective <- c(
   "first-class",
   "first-rate",
   "flawless",
+  "formidabel"
```

Comment:
Did you mean to add "formidable"? And can you please add a comma at the end? Thanks!